package.json

```diff
@@ -51,6 +51,7 @@
   "devDependencies": {
     "@hapi/hapi": "18.3.2",
     "@hapi/joi": "15.1.1",
+    "chance": "^1.0.18",
```

Comment:
please remove caret

Reply:
and updated version of the package.json

Reply:
```suggestion
```

Reply:
Welp I removed it then had to re-add. What do you mean "and updated version of the package.json"?